engine-clients/src/main/java/io/nosqlbench/engine/clients/grafana/analyzer/GrafanaRegionAnalyzer.java

```diff
@@ -0,0 +1,194 @@
+    }
+
+    public GSnapshotInfo createSnapshotForAnnotation(GAnnotation anno, GDashboard dashboard, String snid) {
+
+        //session: scenario_20201215_050435_240
+        //[2020-12-15T05:04:37.232Z[GMT] - 2020-12-15T05:04:37.232Z[GMT]]
+        //span:interval
+        //details:
+        // params: ActivityDef:(4)/{keycount=5000000000L, hosts=node1, main-cycles=500, threads=1, workload=./keyvalue.yaml, cycles=2, stride=2, tags=phase:schema, password=cassandra, rf=3, pooling=16:16:500, driver=cql, rampup-cycles=5000000000, alias=keyvalue_default_schema, valuecount=5000000000L, errors=count, username=cassandra}
+        //labels:
+        // layer: Activity
+        // alias: keyvalue_default_schema
+        // driver: cql
+        // workload: ./keyvalue.yaml
+        // session: scenario_20201215_050435_240
+        // span: interval
+        // appname: nosqlbench
+
+        //    "time": {
+        //      "from": "2020-12-15T04:19:54.943Z",
+        //      "raw": {
+        //        "from": "2020-12-15T04:19:54.943Z",
+        //        "to": "2020-12-15T15:25:11.743Z"
+        //      },
+        //      "to": "2020-12-15T15:25:11.743Z"
+        //    },
+
+        ZoneId zoneid = ZoneId.of("GMT");
+        ZonedDateTime startTime = ZonedDateTime.ofInstant(Instant.ofEpochMilli(anno.getTime()), zoneid);
+        ZonedDateTime endTime = ZonedDateTime.ofInstant(Instant.ofEpochMilli(anno.getTimeEnd()), zoneid);
+        String startIsoInstant = startTime.format(DateTimeFormatter.ISO_INSTANT);
+        String endIsoInstant = endTime.format(DateTimeFormatter.ISO_INSTANT);
+
+        dashboard.getTime().setFrom(startIsoInstant);
+        dashboard.getTime().setTo(endIsoInstant);
+
+        GSnapshotInfo snapshotInfo = gclient.createSnapshot(dashboard, snid);
+        return snapshotInfo;
+    }
+
+    public void getQueries(GDashboard db) {
+
+        List<GDataSource> datasources = getClient().getDatasources();
+
+        List<GPanelDef> mainpanels = db.getPanels().stream()
+                .filter(p -> p.getType().equals("graph"))
+                .collect(Collectors.toList());
+
+        Map<String, Set<String>> tplValues = getClient().resolveAllTplValues(
+                db.getTemplating().getList(),
+                db.getTime().getFrom(),
+                db.getTime().getTo());
+        GStitcher stitcher = new GStitcher(tplValues);
+
+
+        for (GPanelDef mainpanel : mainpanels) {
+            long id = mainpanel.getId();
+            String title = mainpanel.getTitle();
+            String description = mainpanel.getDescription();
+
+            List<GPanelDef> panels = mainpanel.getPanels();
+            String datasource = mainpanel.getDatasource();
+            Map<String, Object> fieldConfig = mainpanel.getFieldConfig();
+            Map<String, String> options = mainpanel.getOptions();
+            List<GPanelDef.GTarget> targets = mainpanel.getTargets();
+            System.out.println("targets:\n" + targets);
```

Comment:
Do we want the Logger usage here?
There are quite a few system.out entries in this file.